### ISSUE TITLE
Themes I would like to see in godot

### ISSUE THREAD

**Issue by Shadowblitz16** [godotengine/godot#25809]
I have a few suggestions on themes I would like to see for godot with syntax highlighting for code

--not sure if these could get godot sued
Visual Studio 2017 Light Theme 
Visual Studio 2017 Blue Theme
Visual Studio 2017 Dark Theme

--these might be hard since they would use textures
PixelArt Light                              
PixelArt Dark
PixelArt Orange and Faded Blue - faded blue is the background color orange is things like buttons and panels

--basically Black and specified color   
Command Line Green                  
Command Line White

I feel like they would make godot more customizable.
I would probably use Visual Studio 2017 Dark Theme and PixelArt Orange and Faded Blue to be honest.

**Comment by mrcdk** [comment]
there are some of those syntax themes already here https://github.com/Calinou/godot-syntax-themes

**Comment by DeadlyLampshade** [comment]
> these might be hard since they would use textures

What you're saying is not actually that hard because you **can** define textures in godot for custom themes. You would need to either store the textures in the theme resource or in relative paths however.
![image](https://user-images.githubusercontent.com/29586664/52611447-60e25900-2ed1-11e9-94ce-635a2bb4dbed.png)

**Comment by akien-mga** [comment]
Syntax themes can be distributed through the asset library, no need to add more to the editor out of the box, unless there's a *strong* need for them.
